pkg/cmd/auth/login/login_test.go

```diff
@@ -18,6 +19,21 @@ import (
 	"github.com/stretchr/testify/assert"
 )
 
+func stubHomeDir(t *testing.T, dir string) {
```

Comment:
What are your thoughts on this function automatically using `t.TempDir()` and returning the directory string so it can be used in the caller? It is less configurable but off the top of my head I can't think of a situation where `t.TempDir()` wouldn't want to be used since that is baked right into the go testing package.

Reply:
I'd prefer to keep it explicit which dir is going to be used as a new home, so I don't mind passing `t.TempDir()` manually. A different kind of test might want to use a different dir!